### PR TITLE
Let exokit-macos take arguments

### PR DESCRIPTION
**Pull request by shawwn**

### CODE REVIEW

File: exokit-macos [hunk 1]
```diff
@@ -14,4 +14,4 @@ if [ -x "$DIR/node/bin/node" ]; then
 fi
 
 cd "$DIR"
-exec "$NODE_BIN" "$DIR/index.js" -h
```
Comment (modulesio) [inline exokit-macos]: Couldn't this just be `-h "$@"`? That's what the other places use I think.

Comment (shawwn) [reply exokit-macos]: No, that would always pass in -h.

**Comment by shawwn** [comment]
It’s probably a better idea to refactor this into exokit.sh and use that instead, since it’s identical code.

**Comment by modulesio** [comment]
Hm, yeah; the only reason it was done in this script instead is because of MacOS `.app` requirements for what's considered a "binary" -- and it can't be called `.sh`.

I'll patch this into both scripts. Thanks!

**Pull request closed without merging**